python/CMakeLists.txt

```diff
@@ -0,0 +1,67 @@
+
+IF(NOT Kokkos_ENABLE_PYBIND11)
+    return()
+ENDIF()
+
+
+# don't build a static python module
+SET(BUILD_SHARED_LIBS ON)
```

Comment:
You already add the library as `MODULE` type below. I don't think this is needed.

Reply:
It has to do with the logic in the FindPython3.cmake module